share/spice/people_in_space/people_in_space.js

```diff
@@ -40,6 +40,7 @@
             signal: "high",
             data: api_result,
             meta: {
+                primaryText: api_result.number + (api_result.number == 1 ? " Person" : " People") + ' in Space',
```

Comment:
I guess we could use `DDG.plural(api_result.number, "Person", "People")` but this is fine.